doc/cephfs/fs-volumes.rst

```diff
@@ -97,6 +97,44 @@ The CephX IDs authorized to <vol_name> need to be reauthorized to <new_vol_name>
 on-going operations of the clients using these IDs may be disrupted. Mirroring is
 expected to be disabled on the volume.
 
+Fetch the information of a CephFS volume using::
+
+    $ ceph fs volume info <vol_name>
+    {
+        "mon_addrs": [
+            "192.168.1.7:40545"
+        ],
+        "pending_subvolume_deletions": 0,
+        "pools": {
+            "data": [
+                {
+                    "avail": 106288734208,
+                    "name": "cephfs.a.data",
+                    "used": 12288
+                }
+            ],
+            "metadata": [
+                {
+                    "avail": 106288734208,
+                    "name": "cephfs.a.meta",
+                    "used": 258048
+                }
+            ]
+        },
+        "size": 124,
+        "subvolumegroups": 1,
+        "volume_name": "a"
+    }
+
+The output format is json and contains fields as follows.
+
+* volume_name: display the CephFS volume name
+* pools: data and metadata pools related information
+* mon_addrs: list of monitor addresses
+* size: current used size of the CephFS volume
```

Comment:
What's the unit for size? From the above output "124?". Its crucial to mention this to avoid guess work.

Reply:
Should I add units `(B)` for bytes in command output or just documenting it is enough.

Reply:
Ideally, both. Better if you can TiB, GiB, etc.. depending on the size.